app/routes/application.js

```diff
@@ -1,13 +1,20 @@
-import Ember from "ember";
+import Route from '@ember/routing/route';
 
-export default Ember.Route.extend({
+import JSTree from 'ember-cli-jstree/components/ember-jstree';
```

Comment:
I do not understand why you need this here. Is it because you import jQuery in index.html ?